splunkoncall.json

```diff
@@ -15,28 +15,28 @@
     "latest_tested_versions": [
         "Cloud. API v1 tested on 2021-02-25"
     ],
-    "app_version": "2.1.1",
+    "app_version": "2.2.0",
     "utctime_updated": "2022-01-28T20:58:55.000000Z",
-    "package_name": "phantom_victorops",
+    "package_name": "phantom_splunkoncall",
     "main_module": "splunkoncall_connector.py",
     "min_phantom_version": "5.1.0",
     "app_wizard_version": "1.0.0",
     "configuration": {
         "api_id": {
-            "description": "API ID from the VictorOps Portal",
+            "description": "API ID from the Splunk On-Call Portal",
             "data_type": "string",
             "required": true,
             "order": 0
         },
         "api_key": {
-            "description": "API Key from the VictorOps Portal",
+            "description": "API Key from the Splunk On-Call Portal",
             "data_type": "password",
             "required": true,
             "order": 1
         },
         "integration_url": {
-            "description": "Custom REST based Integration URL defined in VictorOps",
-            "verbose": "Populate this field if you plan to create, update, or resolve incidents. i.e https://alert.victorops.com/integrations/generic/20131114/alert/84d34810-7b78-4cb9-91a4-3425aeecce68",
+            "description": "Custom REST based Integration URL defined in Splunk On-Call",
+            "verbose": "Populate this field if you plan to create, update, or resolve incidents. i.e https://alert.victorops.com/integrations/generic/20131114/alert/84d34810-7b78-4cb9-91a4-3425aeecce68.",
```

Comment:
are the URLs also changing from alert.victorops.com to alert.splunkoncall.com?

Reply:
We have checked there is no change in the URL so we kept it as it is.